spec/classes/pulpcore_spec.rb

```diff
@@ -19,6 +19,22 @@
             .that_notifies(['Class[Pulpcore::Database]', 'Class[Pulpcore::Service]'])
         end
       end
+
+      context 'with external postgresql' do
+        let :params do
+          {
+            postgresql_db_host: '222.222.222.222',
```

Comment:
https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600

Reply:
Bookmarked, thanks